### PR TITLE
made booking views of index, new, edit and show

### PR DESCRIPTION
**Pull request by chihoko12**
I roughly made booking views.

### CODE REVIEW

File: app/views/bookings/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<%= simple_form_for(restaurant) do |f| %>
```
Comment (rsmml) [inline app/views/bookings/_form.html.erb:1]: isn't it
`<%= simple_form_for [@user, @boat, @booking] do |f| %>` ?
We don't have a restaurant 😄 
And the bookings belongs to users and boats

File: app/views/bookings/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<h2>Here is your bookings</h2>
```
Comment (rsmml) [inline app/views/bookings/index.html.erb:1]: Here are your bookings

File: app/views/bookings/_form.html.erb (hunk 1 above)
Comment (rsmml) [reply app/views/bookings/_form.html.erb:1]: Like when we needed to make a new dose for mr cocktail. The simple form was `[@cocktail, @dose]`

Comment (chihoko12) [reply app/views/bookings/_form.html.erb]: oops. I blindly copied and pasted the code. Thank you for having noticed it. I corrected it.

File: app/views/bookings/index.html.erb (hunk 2 above)
Comment (chihoko12) [reply app/views/bookings/index.html.erb]: thank you. I corrected it.

File: app/views/bookings/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<%= simple_form_for [@user, @boat, @booking] do |f| %>
```
Comment (sorina-duca) [inline app/views/bookings/_form.html.erb:1]: I believe you cannot have three objects in the array, only two. maybe we check with a TA

**Comment by rsmml** [comment]
Looks nice

**Pull request merged**